### ISSUE TITLE
Update check issue

### ISSUE THREAD

**Issue by kratz00** [john30/ebusd#617]
### Description

The official binary build reports the following version number:

```
ebusd --version
ebusd 22.3.p20220508
```
It looks like the update check expects the revision to be equal to the related git tag of the released ebusd version (22.3 and v22.3 for example).

While building the official release binaries git describe fails and instead the *date* fallback is used
- https://github.com/john30/ebusd/blob/7e16af8282aba05eae6a2be420b0bc1e3da7c7dc/configure.ac#L140
- https://github.com/john30/ebusd/blob/a8335ca24a30380151f18dacd6d545dc07934c0b/CMakeLists.txt#L37

 Which then leads to a false positive update check.

### Actual behavior

Running the latest official binaries returns *revision v22.3 available*

### Expected behavior

Running the latest official binaries should return *OK*

### ebusd version

22.3

### ebusd arguments

-v

### Operating system

Debian 11 (Bullseye) / Raspbian 11 / Ubuntu 20-21

### CPU architecture

x64

### Hardware interface

adapter 3 WiFi

### Related integration

TCP (cmdline client like ebusctl or netcat)

### Logs

update check: revision v22.3 available

**Comment by jordipalet** [comment]
I believe I've a similar problem. When I look at data?def, I can see a shorter than usual result and at the end:

 "global": {
  "version": "21.3.v21.3",
  "updatecheck": "version 22.3 available",
  "signal": true,
  "symbolrate": 59,
  "maxsymbolrate": 205,
  "qq": 49,
  "reconnects": 0,
  "masters": 5,
  "messages": 4,
  "lastup": 1657656686
 }

How can I force the update?

(running on Raspbian)

**Comment by jordipalet** [comment]
> I believe I've a similar problem. When I look at data?def, I can see a shorter than usual result and at the end:
> 
> "global": { "version": "21.3.v21.3", "updatecheck": "version 22.3 available", "signal": true, "symbolrate": 59, "maxsymbolrate": 205, "qq": 49, "reconnects": 0, "masters": 5, "messages": 4, "lastup": 1657656686 }
> 
> How can I force the update?
> 
> (running on Raspbian)

Resolved, changed the config file (should be the default maybe?) to --updatecheck=on

**Comment by kratz00** [comment]
Fixed with v22.4, thank you very much John.

```
ebusd --version
ebusd 22.4.v22.4
```
